code/modules/reagents/chemistry/reagents.dm

```diff
@@ -1,4 +1,7 @@
 #define REM REAGENTS_EFFECT_MULTIPLIER
+#define METABOLITE_RATE     0.5 // How much of a reagent is converted metabolites if one is defined
+#define MAX_METABOLITES		15  // The maximum amount of a given metabolite someone can have at a time
+#define METABOLITE_PENALTY(path) min(max(M.reagents.get_reagent_amount(path)/2.5, 1), 5) //Ranges from 1 to 5 depending on level of metabolites. 
```

Comment:
```suggestion
#define METABOLITE_PENALTY(path) clamp(M.reagents.get_reagent_amount(path)/2.5, 1, 5) //Ranges from 1 to 5 depending on level of metabolites. 
```

clamp() proc is identical to the mix of min(max())

> This is effectively equivalent to min(max(Number, Low), High), but with some slight differences. For one thing, clamp() is forgiving if you accidentally swap Low and High; it will just swap them back so Low is always lower. Also, because this is a single proc call it's slightly faster.
~DM docu

Reply:
Ah, thanks much.